packages/snaps-sdk/src/jsx/components/Skeleton.ts

```diff
@@ -0,0 +1,40 @@
+import { createSnapComponent } from '../component';
+
+/**
+ * Definition of Skeleton border radius.
+ */
+export type SkeletonBorderRadius = 'none' | 'medium' | 'full' | undefined;
```

Comment:
Since this is the same as the image radius, maybe extract this to a shared place as `BorderRadius`, and re-use it in both places?

Reply:
It's merged:
- #3023